tla/raft_spec/README.md

```diff
@@ -0,0 +1,39 @@
+# TLA+ model of CCF's implementation of Raft
+
+The given model consists of three files:
+
+- `ccfraft.tla` : The core specification that implements the features as seen below.
+- `MCraft.tla` : The model checking implementation for the spec. Sets the constants and can be modified for each run to
+  fine tune the settings.
+- `MCraft.cfg` : The core configuration which invariants are to be checked etc. Usually stays untouched during normal
+  model checking.
+
+To run this spec, run the `MCraft.tla` with the `-deadlock` parameter to disable liveness checking (see explanation
```

Comment:
Would it be worth having a simple `run.sh` script in the repo as well?

Reply:
Good idea, I added scripts in the tla folder. The reduced spec should run in a few seconds, can you try to run it and report back whether it was usable and documented? Maybe both @achamayou and @jumaffre

Reply:
Tested locally and it worked fine on my local VM. Note that I had to install Java beforehand (`$ sudo apt install default-jre`).